### PR TITLE
Mui fine foods stores page reviews resolved

### PR DESCRIPTION
**Pull request by ozkalai**
Test me! 'MUI'
[Link to MUI-FINE-FOODS-STORE2](https://mui-fin-refine.pankod.com)

Test me! 'MUI'
[Link to MUI-FINE-FOODS-STORE2](https://mui-fin-client.refine.pankod.com)

Fixing the reviews on the stores' page that prepared for the fine food example.

### CODE REVIEW

File: examples/fineFoods/admin/mui/src/pages/stores/create.tsx [hunk 1]
```diff
@@ -152,14 +141,22 @@ export const StoreCreate: React.FC<IResourceComponentsProps> = () => {
...
+                                    required
+                                >
                                     {t("products.fields.isActive")}
                                 </FormLabel>
                                 <Controller
                                     control={control}
                                     name="isActive"
                                     defaultValue={""}
-                                    rules={{ required: true }}
+                                    rules={{ required: "Selection required" }}
```
Comment (omeraplak) [inline examples/fineFoods/admin/mui/src/pages/stores/create.tsx:159]: translate

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Review by salihozdemir — APPROVED** [review]

File: packages/mui/src/components/crud/edit/index.tsx [hunk 2]
```diff
@@ -97,13 +99,17 @@ export const Edit: React.FC<EditProps> = ({
...
             <CardHeader
-                title={translate(
-                    `${resource.name}.titles.edit`,
-                    `Edit ${userFriendlyResourceName(
-                        resource.label ?? resource.name,
-                        "singular",
-                    )}`,
-                )}
+                title={
+                    <Typography sx={{ fontSize: "20px", fontWeight: 600 }}>
```
Comment (omeraplak) [inline packages/mui/src/components/crud/edit/index.tsx:103]: Wouldn't it be better to use `h5` instead? as in the original, https://github.com/mui/material-ui/blob/master/packages/mui-material/src/CardHeader/CardHeader.js#L95

Comment (ozkalai) [reply packages/mui/src/components/crud/edit/index.tsx:103]: make sense :)

Comment (ozkalai) [reply packages/mui/src/components/crud/edit/index.tsx:103]: Do you think we should set a specific font weight ( 600 ) for the "h5" variant on Theme in the future? If not I am gonna pass as a prop.

**Review by omeraplak — APPROVED** [review]

**Pull request merged**